assets/js/domain-verifier-checker.js

```diff
@@ -0,0 +1,179 @@
+//Browserify was used to create domain-verifier-bundle.js so that we can use 'require' in the browser
+
+const codec = require("ripple-binary-codec");
+const addressCodec = require("ripple-address-codec");
+const keyCodec = require("ripple-keypairs");
+
+const TIPS =
+  '<p>Check if the file is actually hosted at the URL above, check your server\'s HTTPS settings and certificate, and make sure your server provides the required <a href="xrp-ledger-toml.html#cors-setup">CORS header.</a></p>';
```

Comment:
It looks like this warning is copy-pasted from the toml checker JS file. I have a feeling it's not as appropriate in this context.

Reply:
Ah yup it needs to be slightly modified for this tool. Thanks for catching.